tests/many_to_one_null/tests.py

```diff
@@ -138,5 +138,6 @@ def test_related_null_to_field(self):
         c1 = Car.objects.create()
         d1 = Driver.objects.create()
         self.assertIs(d1.car, None)
-        with self.assertNumQueries(0):
-            self.assertEqual(list(c1.drivers.all()), [])
+        msg = '"<Car: Car object (1)>" needs to have a value for field "make" before this reverse many-to-one relationship can be used'
+        with self.assertRaisesMessage(ValueError, msg):
+            c1.drivers.all()
```

Comment:
Why this error is raised? This should return an empty list without raising an exception.

Reply:
I have put comment in ticket about that.